<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginLauncherClass</class>
 <widget class="QWidget" name="LoginLauncherClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginLauncher</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="HBoxLayout1">
    <item>
     <widget class="QLabel" name="userLabel">
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="userLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="HBoxLayout2">
    <item>
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>密  码：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>241</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LoginLauncher.qrc"/>
 </resources>
 <connections/>
</ui>
